app/controllers/katello/concerns/api/v2/bulk_hosts_extensions.rb

```diff
@@ -17,7 +17,7 @@ def find_bulk_hosts(permission, bulk_params, restrict_to = nil)
 
         if bulk_params[:included][:search]
           search_hosts = ::Host::Managed.authorized(permission)
-          search_hosts = search_hosts.where(:organization_id => organization_id) if params[:organization_id]
```

Comment:
Curious about this. Was organization_id an undefined variable? are we missing test coverage here? We should add it :)

Reply:
Yeah!  Maybe I'm missing something major, but as far as I could tell `organization_id` is not defined and would just return `nil` always.  And it probably went unchallenged because none of the `bulk_hosts_extensions` tests call `find_bulk_hosts` with an `organization_id` param.  The controller actions just pass through params as-is, so my guess is the only time we'd run into this bug is if a user were making API requests without using the web UI or Hammer.

Reply:
Dang. Did you run into an error when you changed the tests, or did you just happen to notice?

Reply:
I just ran into it when I was working on the tests.  It doesn't even try to evaluate that variable unless `params[:organization_id]` is truthy -- which is one of the things I had to change for the tests.

Reply:
Cool. bonus bug fix! we must not be using the included[:search] path much if at all since it looks like this has been broken since 2016

Reply:
Now that I'm [initializing](https://github.com/Katello/katello/blob/7e57aab509b71c837fec6012e7d2f61329c521f7/test/controllers/api/v2/concerns/bulk_hosts_extensions_test.rb#L32) the `TestController` to send `params[:organization_id]` with each test,
```ruby
@controller = TestController.new(organization_id: @organization.id)
```
I think that should take care of giving us test coverage on the problematic line.

Reply:
(I had added a test but realized it's not needed because ^^)

Reply:
Gotcha. I think we're good to go w.r.t that!